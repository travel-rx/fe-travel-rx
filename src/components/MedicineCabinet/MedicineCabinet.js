import React, { Component } from 'react';
import { StyleSheet, Text, View, TextInput, TouchableOpacity } from 'react-native';
import Footer from '../Footer/Footer';
import { MaterialCommunityIcons } from '@expo/vector-icons';

export default class MedicineCabinet extends Component {
  
  static navigationOptions = {
    title: 'Medicine Cabinet',
    headerStyle: {
      backgroundColor: '#3499AA'
    },
    headerTintColor: '#EBEBEB',
    headerTitleStyle: {
      fontSize: 30,
    },
  };
  
  render() {
    const { navigation } = this.props;
    
    return (
      <View style={styles.container}>
        {/* <Text style={styles.heading}>Medicine Cabinet</Text> */}
        <View style={styles.medContainer}>
          <View style={styles.medicine}>
            <MaterialCommunityIcons
              color='#3499AA'
              name='pill'
              size={65}
              />
              <View>
                <Text style={styles.name}>Phenergan</Text>
                <Text style={styles.genericName}>Promethazine</Text>
              </View>
          </View>
          <View style={styles.medicine}>
            <MaterialCommunityIcons
              color='#3499AA'
              name='pill'
              size={65}
              />
              <View>
                <Text style={styles.name}>Vibramycin</Text>
                <Text style={styles.genericName}>Doxycycline</Text>
              </View>
          </View>
        </View>
        <TouchableOpacity style={styles.button}>
          <Text style={styles.buttonText}>
            Add Medication
          </Text>
        </TouchableOpacity>
        <Footer navigation={navigation}/>      
      </View>
    )
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'space-between'
  },
  heading: {
    color: '#3499AA',
    fontSize: 40,
    paddingTop: 20,

  },
  buttonText: {
    color: '#EBEBEB',
    textAlign: 'center',
    fontSize: 18,
    fontWeight: 'bold',
  },
  medContainer: {
    flex: 1,
    margin: 25
  },
  medicine: {
    flex: 1,
    flexDirection: 'row'
  },
  name: {
    color: '#3499AA',
    fontSize: 35,
    fontWeight: 'bold'
  },
  genericName: {
    fontSize: 15,
  },
  button: {
    backgroundColor: '#8BC5C5',
    borderColor: 'white',
    borderWidth: 1,
    borderRadius: 12,
    display: 'flex',
    fontWeight: 'bold',
    height: 45,
    justifyContent: 'center',
    overflow: 'hidden',
    padding: 0,
    width: 250,
    marginBottom: 25
  },
  login: {
    height: 400,
    flex: 0,
    justifyContent: 'space-evenly'
  }
})
