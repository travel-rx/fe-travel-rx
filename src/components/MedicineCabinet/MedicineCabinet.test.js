import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import { MedicineCabinet } from './MedicineCabinet';
import { shallow } from 'enzyme';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('MedicineCabinet', () => {
  it('should return true', () => {
    expect(true).toEqual(true);
  })
})