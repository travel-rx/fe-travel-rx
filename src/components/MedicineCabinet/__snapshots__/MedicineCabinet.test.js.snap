// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MedicineCabinet should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "space-between",
    }
  }
>
  <ComponentWrapper>
    <TouchableOpacity
      activeOpacity={0.2}
      key="Phenergan"
      onPress={[Function]}
      style={
        Object {
          "flex": 1,
          "flexDirection": "row",
          "marginTop": 80.03999999999999,
          "width": 675,
        }
      }
    >
      <Icon
        allowFontScaling={false}
        color="#3499AA"
        name="pill"
        size={50}
      />
      <View>
        <Text
          style={
            Object {
              "color": "#3499AA",
              "fontSize": 53.36,
              "fontWeight": "bold",
              "marginLeft": 15,
            }
          }
        >
          Phenergan
        </Text>
        <Text
          style={
            Object {
              "color": "#212B31",
              "fontSize": 40.019999999999996,
              "marginLeft": 15,
            }
          }
        >
          Promethazine
        </Text>
      </View>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
      key="Vibramycin"
      onPress={[Function]}
      style={
        Object {
          "flex": 1,
          "flexDirection": "row",
          "marginTop": 80.03999999999999,
          "width": 675,
        }
      }
    >
      <Icon
        allowFontScaling={false}
        color="#3499AA"
        name="pill"
        size={50}
      />
      <View>
        <Text
          style={
            Object {
              "color": "#3499AA",
              "fontSize": 53.36,
              "fontWeight": "bold",
              "marginLeft": 15,
            }
          }
        >
          Vibramycin
        </Text>
        <Text
          style={
            Object {
              "color": "#212B31",
              "fontSize": 40.019999999999996,
              "marginLeft": 15,
            }
          }
        >
          Doxycycline
        </Text>
      </View>
    </TouchableOpacity>
  </ComponentWrapper>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
    style={
      Object {
        "backgroundColor": "#3499AA",
        "borderColor": "white",
        "borderRadius": 12,
        "borderWidth": 1,
        "display": "flex",
        "fontWeight": "bold",
        "height": 45,
        "justifyContent": "center",
        "marginBottom": 66.7,
        "overflow": "hidden",
        "padding": 0,
        "width": 637.5,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#EBEBEB",
          "fontSize": 53,
          "textAlign": "center",
        }
      }
    >
      Add Medication
    </Text>
  </TouchableOpacity>
  <Footer />
</View>
`;
